src/__experimental__/components/decimal/index.js

```diff
@@ -0,0 +1,3 @@
+import Decimal from './decimal.component';
```

Comment:
Is there a reason that we can't use? 
```js
export { Decimal } from './decimal.component';
```

Reply:
That'll be because the Decimal component is the `default` export from that file.